modules/RTC/JitsiTrack.js

```diff
@@ -134,7 +140,7 @@ export default class JitsiTrack extends EventEmitter {
             }
         }
         if (this._streamInactiveHandler) {
-            addMediaStreamInactiveHandler(this.stream, undefined);
+            this._addMediaStreamInactiveHandler(undefined);
```

Comment:
this is a bit confusing, why isn't the streamInactiveHandler passed here as well?

Reply:
we are unregistering the handler here when the stream is getting removed from the conference.

Reply:
ah. and it uses .onended, not addEventListener('ended') so that works. :+1: